### ISSUE TITLE
WebUI: renaming 'Flow params' results in invisible Flow configurations

### ISSUE THREAD

**Issue by JGrotex** [TIBCOSoftware/flogo#102]
renaming 'Flow params' results in invisible Flow configurations, see code section containing two attribute mappings in the Simple Return Activity.  WebUI tester shows no Issue, but if you run with the build App the following Error is shown:

> 2017-12-06 09:47:14.923 ERROR  [engine] - Execution failed for Activity[Return] in Flow[validateIBAN] - attribute not in scope
> 2017-12-06 09:47:14.923 ERROR  [model-tibco-simple] - Error evaluating activity 'Return'[github-com-tibco-software-flogo-contrib-activity-actreturn] - attribute not in scope

Reproduce:

1. define 'Flow params' Output Parameter "feedback" 
2. go to Simple Return Activity and map a value
3. rename 'Flow params' Output Parameter "feedback" to "feedback2"
4. go to Simple Return Activity and map a value

^ in 4. the field "feedback" is invisible inside the export, so the Issue can not be fixed inside the WebUI.
You have to export the App, fix the JSON and re-Import the App.

Result Sample containing mapping of "feedback" and "feedback2"
```
"attributes": [
                  {
                    "name": "mappings",
                    "value": [
                      {
                        "mapTo": "feedback",
                        "type": 4,
                        "value": {
                          "result": "{{$activity[rest_2].result}}"
                        }
                      },
                      {
                        "mapTo": "feedback2",
                        "type": 1,
                        "value": "$flow.iban"
                      }
                    ],
                    "required": false,
                    "type": "array"
                  }
                ]
```

**Comment by JGrotex** [comment]
**_Workaround using WebUI for Info,_** you can fix this Issue using the WebUI by renaming the Field to the old name remove the mapping in the Simple Return Activity and rename the Field again to the new name.

**Comment by mellistibco** [comment]
Resolved in 0.5.1.. Latest on Docker Hub
